templates/partials/configuration_upgrade.html

```diff
@@ -0,0 +1,77 @@
+<div class="upgrade-config col-sm-12" ng-hide="deployInfo">
+  <legend translate>instance.upgrade.no_horti</legend>
+  <p translate>instance.upgrade.no_horti.detail</p>
+</div>
+<div class="upgrade-config col-sm-12" ng-show="deployInfo">
+  <div ng-show="loading || upgrading">
+    <div class="loader"></div>
+    <p class="status" ng-show="upgrading"><span translate>instance.upgrade.upgrading</span></p>
+  </div>
+
+  <div ng-show="error">
+    <p class="error">{{error}}</p>
+  </div>
+
+  <div ng-hide="loading || upgrading">
+
+    <div>
+      <legend translate>instance.upgrade.current_version</legend>
+      <p ng-hide="deployInfo" translate>instance.upgrade.no_details</p>
+      <dl ng-show="deployInfo" class="horizontal">
+        <dt translate>instance.upgrade.version</dt>
+        <dd>{{deployInfo.version}}</dd>
+        <dt translate>instance.upgrade.deployed_by</dt>
+        <dd>{{deployInfo.user}}</dd>
+        <dt translate>instance.upgrade.at</dt>
+        <dd>{{deployInfo.timestamp | date}}</dd>
+      </dl>
+    </div>
+
+    <div>
+      <legend translate>instance.upgrade.releases</legend>
+      <p ng-hide="versions.releases.length" translate>instance.upgrade.no_new_releases</p>
+      <ul ng-show="versions.releases.length" class="table table-striped">
+        <li ng-repeat="v in versions.releases" class="row">
+          <div class="col-xs-5"><code>{{v.id}}</code></div>
+          <div class="col-xs-4">{{v.value.build_time | date}}</div>
+          <div class="col-xs-3"><a ng-click="upgrade(v.id)">Upgrade</a></div>
```

Comment:
Internationalisation - the other two blocks are correct, this one must've been missed!